iree_tests/benchmarks/benchmark_sdxl_rocm.py

```diff
@@ -0,0 +1,63 @@
+import os
+from collections import namedtuple
+
+benchmark_dir = os.path.dirname(os.path.realpath(__file__))
+
+def launchCommand(scriptcommand):
+    print("Launching:", scriptcommand, "[ Proc:", os.getpid(), "]")
+    try:
+        ret = os.system(scriptcommand)
+        return ret
+    except OSError as errormsg:
+        print(
+            "Invoking ",
+            scriptcommand,
+            " failed:",
+            errormsg,
+        )
+        return 1
+
+def run_sdxl_rocm_benchmark():
+    scriptcommand = (
+        "bash "
+        + f"{benchmark_dir}/benchmark_sdxl_rocm.sh"
+    )
+    return launchCommand(scriptcommand)
+
+BenchmarkResult = namedtuple(
+    "BenchmarkResult", "benchmark_name time cpu_time iterations user_counters"
+)
+
+def decode_output(bench_lines):
+    benchmark_results = []
+    for line in bench_lines[3:]:
```

Comment:
Oh lol, yeah - if you're going to do this sort of parsing, just have the benchmark output a machine readable format instead :P